01_temperature_processing.R

```diff
@@ -75,7 +80,8 @@ logchar<-unique(audits[,c('LOGGER_ID','PARAMETER')])
 names(logchar)<-c('log','char')
 
 # Load the QC criteria for continuous data
-ConQC <- read.csv('//deqlab1/wqm/Volunteer Monitoring/datamanagement/R/ContinuousDataReview/ConQC.csv')
+load('//deqlab1/wqm/Volunteer Monitoring/datamanagement/R/ContinuousDataReview/ConQC.RData')
```

Comment:
What is the best way to keep this in the project?


Steve Hanson
ODEQ Laboratory
Ph: 503.693.5737

From: petertbryant [mailto:notifications@github.com]
Sent: Friday, August 05, 2016 9:55 AM
To: petertbryant/ContinuousDataReview
Cc: HANSON Steve; Mention
Subject: Re: [petertbryant/ContinuousDataReview] Comments for grade changes (#3)


In 01_temperature_processing.R<https://github.com/petertbryant/ContinuousDataReview/pull/3#discussion_r73724382>:

> @@ -75,7 +80,8 @@ logchar<-unique(audits[,c('LOGGER_ID','PARAMETER')])

>  names(logchar)<-c('log','char')

>

>  # Load the QC criteria for continuous data

> -ConQC <- read.csv('//deqlab1/wqm/Volunteer Monitoring/datamanagement/R/ContinuousDataReview/ConQC.csv')

> +load('//deqlab1/wqm/Volunteer Monitoring/datamanagement/R/ContinuousDataReview/ConQC.RData')

This file is not in this location currently. Perhaps will all the back and forth with commits it was missed? Since we are working within a project we don't need to use the full path name either, for future reference.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/petertbryant/ContinuousDataReview/pull/3/files/5e0a4e5e7e6880fd411f9290baa0ef06d04cfeb8#r73724382>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATJGv-2eOs_dpVFdP2GS5dJV6VQCArjcks5qc2rQgaJpZM4Jb301>.